packages/app/package.json

```diff
@@ -89,6 +89,7 @@
     ]
   },
   "devDependencies": {
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
```

Comment:
Is this being used?

Reply:
It's hard to catch, but in dev mode there's a warning that the dependency should be explicit.